tests/test_core.py

```diff
@@ -1636,12 +1636,7 @@ def test_exceptions_rethrow_missing(self):
   @no_wasm64('MEMORY64 does not yet support exceptions')
   @with_both_eh_sjlj
   def test_exception_message(self):
```

Comment:
```suggestion
  def test_EXCEPTION_PRINTING_SUPPORT(self):
```
Might be nice to rename the test to match the flag, as it can be seen as a dedicated test for that option now?